t/ui/10-tests_overview.t

```diff
@@ -360,6 +360,18 @@ subtest "job template names displayed on 'Test result overview' page" => sub {
     is(wait_for_element(selector => '.popover-header')->get_text, 'kde_variant', 'description popover shows content');
 };
 
+subtest "job dependencies displayed on 'Test result overview' page" => sub {
+    $driver->get($baseurl . 'tests/overview?distri=opensuse&version=13.1&build=0091&groupid=1001');
+    my $deps              = $driver->find_element('td#res_DVD_x86_64_kde .dependency');
+    my @children_elements = $driver->find_child_elements($deps, 'a');
+    like($children_elements[0]->get_attribute('href'), qr/tests\/99963#dependencies/, 'job href is shown correctly');
+    is(
+        $children_elements[0]->get_attribute('title'),
+        "1 parallel parent\ndependency passed",
+        'dependency is shown correctly'
+    );
```

Comment:
I think we can try to shorten this a bit, prevent line-breaks, simplify and make the qr-escaping easier for URLs, e.g.:

```perl
my $details = $children_elements[0];
like $details->get_attribute('href'), qr{tests/99963#dependencies}, 'job href is shown correctly';
is $details->get_attribute('title'), "1 parallel parent\ndependency passed", 'dependency shown correctly';
```

Reply:
fixed